src/components/Button.js

```diff
@@ -0,0 +1,19 @@
+import PropTypes from 'prop-types';
+
+function Button(props) {
+  const { name } = props;
+  return (
+    <button type="button">
+      {name}
+    </button>
+  );
+}
+
+Button.defaultProps = {
```

Comment:
**[OPTIONAL]**
* [ ] Why set a default props for button name, the result prop in the display has a default prop of '0' because we want the app to default to '0' on reset. I am not too sure what this would achieve.🤔

Reply:
Because the linter was actually asking for it.

I've now added the isRequired rule to that prop and no longer need the defaultProps.

Thanks for pointing it out.